packages/vitaminjs-jest/src/jestrc.js

```diff
@@ -0,0 +1,15 @@
+import { resolveConfigPath, resolveConfigModule } from 'vitaminjs-build/config/resolve';
+
+export default JSON.stringify({
+    rootDir: process.cwd(),
```

Comment:
This is brittle, shouldn't we use one of the `VITAMIN_*` env vars with application root?